<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color:#F0EEEE ;

border-top-left-radius: 6px 6px;
border-top-right-radius: 6px 6px;
border-bottom-right-radius: 6px 6px;
border-bottom-left-radius: 6px 6px;

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="topMargin">
     <number>9</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btAgregar">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>33</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu,Helvetica,sans-serif</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	color: white;
		
	background-color:  rgb(0, 85, 0);

    border: 2px solid rgb(0, 85, 0);
    border-radius:3px;

    padding:3px 3px;
    font:10 bold;
    font-family: Ubuntu, Helvetica, sans-serif;
}
QPushButton:pressed{	
	border: 3px solid #505050;
}</string>
          </property>
          <property name="text">
           <string>Nuevo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">QLabel{
color:rgb(85, 85, 85);
font:bold;
font-family: Ubuntu, Helvetica, sans-serif;
}</string>
          </property>
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QLineEdit" name="leBusqueda">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>96</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit { 
border: 1px solid #999999; 
padding: 2px 2px 2px 2px;
min-width: 90px;
height: 20px;

border-top-left-radius: 3px ;
border-top-right-radius: 0px solid white;
border-bottom-right-radius: 0px ;
border-bottom-left-radius: 3px;

}</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbLupita">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>26</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>26</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel { 
background: white;
border: 1px solid #999999; 
padding: 2px 2px 2px 2px;

height: 30px;
width:26px;

border-top-left-radius: 0px ;
border-top-right-radius: 3px;
border-bottom-right-radius: 3px;
border-bottom-left-radius: 0px;

border-left: 0px solid white;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>:/newPrefix/find.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">QLabel{
color:rgb(85, 85, 85);
font:bold;
font-family: Ubuntu, Helvetica, sans-serif;
}</string>
          </property>
          <property name="text">
           <string>Filtros</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbCampos">
          <property name="styleSheet">
           <string notr="true">QComboBox {
border: 1px solid #999999; 
border-radius: 4px;
padding: 2px 2px 2px 2px;
min-width: 90px;
height: 20px;
}

QComboBox::down-arrow {
image: url(:newPrefix/arrow_down.png);
height: 10px;
}
QComboBox:editable {
background: white;
border-radius: 0px;
}
QComboBox::drop-down { border-style: none}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="4,1">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lbTitulo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>42</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Ubuntu,Helvetica,sans-serif</family>
              <pointsize>-1</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
color: #252424;
background-color:rgb(244, 244, 244);

padding:3px;
font: bold 20px;
font-family: Ubuntu, Helvetica, sans-serif;
border: 1px solid #BFBFBF;

border-bottom: 0px solid white;
border-top-left-radius: 3px ;
border-top-right-radius: 0px solid white;
border-bottom-right-radius: 0px ;

}</string>
            </property>
            <property name="text">
             <string>&lt;Nombre lista&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbCantidadItems">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>42</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
color: rgb(147, 147, 147);
background-color:rgb(244, 244, 244);

padding:3px;
font: 10px;
font-family: Ubuntu, Helvetica, sans-serif;
border: 1px solid #BFBFBF;

border-top-left-radius: 0px solid rgb(244, 244, 244);
border-top-right-radius: 3px;
border-bottom-right-radius: 3px;
border-bottom-left-radius: 0px;
border-bottom: 0px solid white;
}</string>
            </property>
            <property name="text">
             <string>0 items(s) listado(s)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,1">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QTableWidget" name="twDatos">
          <property name="font">
           <font>
            <family>Ubuntu,Helvetica,sans-serif</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QHeaderView::section {
border-radius: 3px;
padding: 2px 4px 2px 4px;

/* Fuente */
color: #3A3939;
text-align: left;
font-family: Ubuntu, Helvetica, sans-serif;

background-color:rgb(244, 244, 244);
/*qlineargradient(spread:reflect, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(240, 240, 240, 255), stop:0.509174 rgba(250, 250, 250, 255), stop:0.963303 rgba(255, 255, 255, 255));*/
/* Bordes */
border-bottom: 1px solid #E3E3E3;
border-top: 1px solid #FFFFFF;
padding:6px;
border-top-left-radius: 0px;

}

QTableWidget{
font-family: Ubuntu, Helvetica, sans-serif;
/*font-size: 12px;*/
text-align: left;

border: 1px solid #BFBFBF;

}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::AnyKeyPressed</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="textElideMode">
           <enum>Qt::ElideLeft</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerItem</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerItem</enum>
          </property>
          <property name="showGrid">
           <bool>true</bool>
          </property>
          <property name="gridStyle">
           <enum>Qt::NoPen</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>24</number>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gbDetalles">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QToolBox" name="toolBox">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QToolBox::tab {
	height:100px;
	color:rgb(68, 68, 68);
	margin-bottom:0px;
	border-bottom: 1px solid rgb(199, 199, 199);
	border-radius: 3px;
	padding:3px;
}
</string>
             </property>
             <property name="tabSpacing">
              <number>0</number>
             </property>
             <widget class="QWidget" name="widget">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>137</width>
                <height>269</height>
               </rect>
              </property>
              <attribute name="label">
               <string>Operaciones </string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QCommandLinkButton" name="btEditar">
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string>Editar</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../GUI/images_rc/icons.qrc">
                   <normaloff>:/newPrefix/edit.png</normaloff>:/newPrefix/edit.png</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCommandLinkButton" name="btEliminar">
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string>Eliminar</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../GUI/images_rc/icons.qrc">
                   <normaloff>:/newPrefix/del.png</normaloff>:/newPrefix/del.png</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Expanding</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images_rc/images.qrc"/>
  <include location="../GUI/images_rc/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cbCampos</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>leBusqueda</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>675</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
